Packs/ForwardXSOARAuditLogsToSplunkHEC/Scripts/ForwardAuditLogsToSplunkHEC/README.md

```diff
@@ -0,0 +1,20 @@
+This Automation script uses the XSOAR API to get the audit logs and pushes them to Splunk HEC. Dependencies: SlunkPy and Demisto REST API integrations
```

Comment:
Why Demisto REST API required? 

If it is really required please add it to `dependencies` field in pack-metadata, as I suggested to add SplunkPy.

Reply:
Demisto API is required to pull the audit logs from the local instance to be forwarded. When contributing from the GUI I cannot set any dependencies hence they were missing.